webpack-plugin/src/apply.js

```diff
@@ -40,7 +41,7 @@ function apply(compiler) {
   }
 
   // Compile the client
-  const userBundleFileName = path.join(dest, 'user-bundle.js');
+  const userBundleFileName = path.join(this.options.dest, 'user-bundle.js');
```

Comment:
Nope, this is used by webpack and the publicPath is prepended by default here (at least when I tested it)